Automattic/wp-calypso
# Help Center: Add GPT-generated response

This adds a GPT-generated response in the Calypso help center, limited to dev, wp-calypso, and staging environments.

![Screen Recording 2023-05-08 at 16 46 08](https://github.com/Automattic/wp-calypso/assets/844866/a69819f3-f404-46d7-8519-c27b8bdb1b5e)


## Testing Instructions

* Navigate to Help Center
* Click to contact via email (or chat if available)
* Type your message
* Pretend to send it
* In the "last chance" screen, you will see the GPT-generated message

## Still to do 

- [ ] Proper ABtesting split so we only show this in some scenarios
- [x] Proper loading indicator
- [x] Maybe feed the links quicker so that we can at least click articles
- [x] Make help articles clickable inside this window
- [x] Add required text mentioning the suggestion is AI-generated.

## Pre-merge Checklist
